docs/choosing_clusters.md

```diff
@@ -43,11 +43,37 @@ In the Docker For Mac preferences, click
 ### Cons
 
 - If Kubernetes breaks, it's easier to reset the whole thing than debug it.
+- Much more resource-intensive because it uses docker-shim as the container runtime.
 - Different defaults than a prod cluster and difficult to customize.
 - Not available on Linux.
 
 ---
 
+## Kind
+
+[Kind](https://kind.sigs.k8s.io/) runs Kubernetes inside a Docker container.
+
+The Kubernetes team uses Kind to test Kubernetes itself. But its fast startup
+time also makes it a good solution for local dev. Follow these instructions for
+setting up Kind with Tilt:
+
+[Kind Setup Instructions](https://github.com/windmilleng/kind-local)
+
+### Pros
+
+- Creating a new cluster is fast (~20 seconds). Deleting a cluster is even faster. 
+- Much more robust than Docker for Mac. Uses containerd instead of docker-shim. Short-lived clusters tend to be more reliable.
+- Supports a local image registry (with our [custom setup instructions](https://github.com/windmilleng/kind-local)).
```

Comment:
hmm, feels weird that this script lives in a separate github repo -- how hard would it be to package this as `tilt kind-cluster`?

Reply:
good question! Two hypotheses:

1) The specifics of this setup are still changing frequently (across Tilt versions and across Kind versions). If we hard-code this in Tilt, that means you have a weird situation where each Tilt version only works with a specific Kind version, and you run into versioning hell (oops, we can't upgrade Tilt to v0.13 because we're still on Kind 0.7)

2) People will have their own cluster setup scripts (i.e., people will want to start up Kind with the particular Kubernetes version they use in production), and will need to know how to modify them to work with Tilt.

Reply:
I wonder if you'd get corresponding problems if we change how we match to the local registry in Tilt and someone has an out-of-date `kind-with-registry.sh`?

Anyway I think we'll need to make this more ergonomic if we want KinD to be Tilt's recommended solution to local dev, but i'm fine doing it this way for now